snakesee/demo/runner.py

```
@@ -0,0 +1,198 @@
+    sleep_min, sleep_max = _DURATION_RANGES[duration]
+    use_plugin = _logger_plugin_available()
+    if not use_plugin:
+        print(
+            "[snakesee demo] snakemake-logger-plugin-snakesee not installed; "
+            "falling back to log-file polling. Install the plugin for realtime events.",
+            file=sys.stderr,
+        )
+
+    argv = _build_snakemake_argv(cores, sleep_min, sleep_max, use_plugin)
+
+    proc: subprocess.Popen[bytes] = subprocess.Popen(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)